config/environments/production.rb

```
@@ -11,7 +11,7 @@
   config.eager_load = true
 
   # Full error reports are disabled and caching is turned on.
-  config.consider_all_requests_local = true
```

Don't think this does what we want.
You can copy this over instead, which will suppress the error messages:
https://github.com/ministryofjustice/staff-device-dns-dhcp-admin/blob/main/public/500.html